File: Banco de dados relacional Postgre/conexao_com_postgre.R
install.packages("RPostgreSQL")
library("RPostgreSQL")

#conectando com o banco de dados
conexao = dbConnect("PostgreSQL", dbname="AulaSQL", host="localhost", port=5432, user="postgres", password="adm")

#consulta SQL
sql = " SELECT QUANTIDADE, VALORTOTAL, PRODUTO, TOTAL FROM ITENSVENDA  IV
      INNER JOIN VENDAS V ON (IV.IDVENDA = V.IDVENDA) 
      INNER JOIN PRODUTOS P ON (IV.IDPRODUTO = P.IDPRODUTO)"
vendas = dbGetQuery(conexao, sql)

#dados retornados
class(vendas)
dim(vendas)
head(vendas)

#encerra a conexão
dbDisconnect(conexao)

File: Banco de dados relacional Postgre/conexao_com_postgre.py
import psycopg2

conexao = psycopg2.connect(host="localhost",database="AulaSQL", user="postgres", password="123456", port=5432)

#cria um cursor
cursor = conexao.cursor()
#define consulta
consulta = "select * from clientes"
#executa
cursor.execute(consulta)

#recupera todos os registros
registros = cursor.fetchall()

#percorre os registros e imprime
for row in registros:
       print("Nome = ", row[1], )
       print("Estado = ", row[2])
       print("Status  = ", row[4], "\n")

#fecha conexão e cursor
cursor.close()
conexao.close()